.github/workflows/website.yml

```diff
@@ -3,7 +3,7 @@ name: website
 on:
   push:
     branches: [main]
-    paths: 
+    paths:
```

Comment:
thanks! did it from GitHub.com 🤪

Reply:
Pro Tip™: Hit `.` on the GitHub page to do it via VSCode Web. If you've got settings sync set up, it can format it via the Prettier extension automatically.

Reply:
ah yes! slipped my mind here 😞